source/macos/AdaptiveCards/AdaptiveCards/Renderers/AdaptiveCardRenderer.swift

```diff
@@ -84,6 +84,10 @@ class AdaptiveCardRenderer {
         
         if let backgroundImage = card.getBackgroundImage(), let url = backgroundImage.getUrl() {
             rootView.setupBackgroundImageProperties(backgroundImage)
+            let padding = CGFloat(truncating: hostConfig.getSpacing()?.paddingSpacing ?? 0)
+            // bleed view setup
+            rootView.setBleedViewConstraint(direction: (true, true, true, true), with: padding)
+            rootView.bleedBackgroundImage(direction: (true, true, true, true), with: padding)
```

Comment:
This is hard to read, better to use a structure

Reply:
Ok